requirements-azure.txt

```diff
@@ -27,7 +27,7 @@ azure-mgmt-nspkg==2.0.0
 azure-mgmt-privatedns==1.0.0
 azure-mgmt-redis==13.0.0
 azure-mgmt-resource==21.1.0
-azure-mgmt-rdbms==10.0.0
+azure-mgmt-rdbms==10.2.0b12
```

Comment:
azure-mgmt-rdbms 10.2.0b12 depends on azure-mgmt-core<2.0.0 and >=1.3.2 . suggeste updating to lastest version(v1.4.0)

Reply:
done